examples/ucp_client_server.c

```diff
@@ -547,6 +584,29 @@ static int parse_cmd(int argc, char *const argv[], char **server_addr,
         case 'i':
             num_iterations = atoi(optarg);
             break;
+        case 's':
+            test_string_length = atol(optarg);
+            if (test_string_length < 0) {
+                fprintf(stderr, "Wrong string size %ld\n", test_string_length);
+                return UCS_ERR_UNSUPPORTED;
+            }	
+            break;
+        case 'm':
+            test_mem_type = parse_mem_type(optarg);
+            if (test_mem_type == UCS_MEMORY_TYPE_LAST) {
+                return UCS_ERR_UNSUPPORTED;
+            }
+            break;
+        case '?':
```

Comment:
getopt may print error message in case if incorrect option or missing parameter... may be just exclude opterr initialization?

Reply:
done